satellitedata/jan_test.csv.dvc

```diff
@@ -0,0 +1,4 @@
+outs:
+- md5: 4a8c9ac0a12753011e9692360fa92951
+  size: 22060747
+  path: jan_test.csv
```

Comment:
qq - what does jan mean in this case? can it be just "test", "train"?

Reply:
**jan** makes reference to the month . January .